### PR TITLE
swagger-petstore v3 sample

### PR DESCRIPTION
**Pull request by ralphdoe**
swagger-petstore v3 sample based on swagger-inflector

### CODE REVIEW

File: java/inflector-petstore-sample/README.MD [hunk 1]
```diff
@@ -0,0 +1,15 @@
+Inflector Jetty WebXML Sample
```
Comment (frantuma) [inline java/inflector-petstore-sample/README.MD]: what is this project? leftover to be removed?

File: java/swagger-petstore/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,12 @@
+FROM openjdk:8-jre-alpine
```
Comment (frantuma) [inline java/swagger-petstore/Dockerfile]: is this part of requirement? do we need a docker image? is this tested?

File: java/swagger-petstore/README.md [hunk 3]
```diff
@@ -0,0 +1 @@
+The new Petstore sample.
```
Comment (frantuma) [inline java/swagger-petstore/README.md]: See readme of other 2.0 inflector based samples and update readme

File: java/swagger-petstore/inflector.yaml [hunk 4]
```diff
@@ -0,0 +1,11 @@
+controllerPackage: io.swagger.petstore.controller
+modelPackage:  io.swagger.petstore.model
+swaggerUrl: ./src/main/swagger/openapi.yaml
+modelMappings:
```
Comment (frantuma) [inline java/swagger-petstore/inflector.yaml]: modelMappings not necessary

File: java/swagger-petstore/src/main/java/io/swagger/petstore/data/PetData.java [hunk 5]
```diff
@@ -0,0 +1,147 @@
+/**
```
Comment (frantuma) [inline java/swagger-petstore/src/main/java/io/swagger/petstore/data/PetData.java]: Either have copyright header in all java files or in none. also cp 2018 not 2016

File: java/swagger-petstore/src/main/swagger/openapi.yaml [hunk 6]
```diff
@@ -0,0 +1,814 @@
...
+        '200':
+          description: Successful operation
+          content:
+            application/xml:
+              schema:
+                $ref: '#/components/schemas/Pet'
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Pet'
+            application/x-www-form-urlencoded:
```
Comment (frantuma) [inline java/swagger-petstore/src/main/swagger/openapi.yaml]: doesn't make sense, remove form-urlencoded responses mediatype wherever found

File: java/swagger-petstore/Dockerfile (hunk 2 above)
Comment (frantuma) [inline java/swagger-petstore/Dockerfile]: is docker part of requirements? if we need, is it tested? if not remove

File: java/swagger-petstore/inflector.yaml (hunk 4 above)
Comment (frantuma) [inline java/swagger-petstore/inflector.yaml]: remove modelMappings as redundant

File: java/swagger-petstore/README.md (hunk 3 above)
Comment (frantuma) [inline java/swagger-petstore/README.md]: update readme, see other 2.0 inflector based samples

File: java/swagger-petstore/src/main/webapp/index.html [hunk 7]
```diff
@@ -0,0 +1,61 @@
+<!-- HTML for static distribution bundle build -->
```
Comment (frantuma) [inline java/swagger-petstore/src/main/webapp/index.html]: use always latest version of ui, as done in other 2.0 samples, eg. java-jersey2-configfile or others, this means keep only updated index.html and modify pom to download ui

File: java/swagger-petstore/src/main/java/io/swagger/petstore/exception/ApiException.java [hunk 8]
```diff
@@ -0,0 +1,26 @@
+/**
```
Comment (frantuma) [inline java/swagger-petstore/src/main/java/io/swagger/petstore/exception/ApiException.java]: either have all java files with copy header (set to 2018, not 2016) or none

File: java/swagger-petstore/inflector.yaml (hunk 4 above)
Comment (ralphdoe) [reply java/swagger-petstore/inflector.yaml]: Done!

File: java/inflector-petstore-sample/README.MD (hunk 1 above)
Comment (ralphdoe) [reply java/inflector-petstore-sample/README.MD]: Removed! It was a sample project. But now everything is in swagger-petstore.

File: java/swagger-petstore/src/main/java/io/swagger/petstore/data/PetData.java (hunk 5 above)
Comment (ralphdoe) [reply java/swagger-petstore/src/main/java/io/swagger/petstore/data/PetData.java]: Done!

File: java/swagger-petstore/src/main/swagger/openapi.yaml (hunk 6 above)
Comment (ralphdoe) [reply java/swagger-petstore/src/main/swagger/openapi.yaml]: Done!

File: java/swagger-petstore/inflector.yaml (hunk 4 above)
Comment (ralphdoe) [reply java/swagger-petstore/inflector.yaml]: Done!

File: java/swagger-petstore/src/main/java/io/swagger/petstore/exception/ApiException.java (hunk 8 above)
Comment (ralphdoe) [reply java/swagger-petstore/src/main/java/io/swagger/petstore/exception/ApiException.java]: Done!

File: java/swagger-petstore/src/main/webapp/index.html (hunk 7 above)
Comment (ralphdoe) [reply java/swagger-petstore/src/main/webapp/index.html]: Done!

File: java/swagger-petstore/README.md (hunk 3 above)
Comment (ralphdoe) [reply java/swagger-petstore/README.md]: Done!

File: java/swagger-petstore/Dockerfile (hunk 2 above)
Comment (ralphdoe) [reply java/swagger-petstore/Dockerfile]: It has been tested! Need to validate if it's necessary.

File: java/swagger-petstore/README.md (hunk 3 above)
Comment (ralphdoe) [reply java/swagger-petstore/README.md]: Done!

File: java/swagger-petstore/Dockerfile (hunk 2 above)
Comment (ralphdoe) [reply java/swagger-petstore/Dockerfile]: It has been tested! Need to validate if it's necessary.

**Pull request merged**